docs(html): Clarify default step base is 0 (1970-01-01) for date inputs (#40365)
mdn/content: files/en-us/web/html/reference/elements/input/date/index.md

### step

> The `step` attribute is a number that specifies the granularity that the value must adhere to, or the special value `any`, which is described below. Only values which are equal to the basis for stepping ([`min`](#min) if specified, [`value`](/en-US/docs/Web/HTML/Reference/Elements/input#value) otherwise, or **`0`** (the Unix epoch, `1970-01-01`) if neither is provided) are valid.

Reviewer: ```suggestion
The `step` attribute is a number that specifies the granularity that the value must adhere to, or the special value `any`, which is described below. Only values which are a whole number of steps from the step base are valid. The step base is [`min`](#min) if specified, [`value`](/en-US/docs/Web/HTML/Reference/Elements/input#value) otherwise, or `0` (the Unix epoch, `1970-01-01`) if neither is provided.
```
